src/main/resources/db/migration/0200/V243__Cohorts.sql

```diff
@@ -0,0 +1,26 @@
+-- Add table that stores accelerator cohort data
+CREATE TABLE cohorts (
+    id BIGINT PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
+    name TEXT NOT NULL UNIQUE,
+    created_by BIGINT NOT NULL REFERENCES users,
+    created_time TIMESTAMP WITH TIME ZONE NOT NULL,
+    modified_by BIGINT NOT NULL REFERENCES users,
+    modified_time TIMESTAMP WITH TIME ZONE NOT NULL
+);
+
+CREATE INDEX ON cohorts (created_by);
+CREATE INDEX ON cohorts (modified_by);
+
+-- Add participant / cohort relationship
+ALTER TABLE participants ADD COLUMN cohort_id BIGINT REFERENCES cohorts;
+CREATE INDEX ON participants (cohort_id);
+
+-- Add cohort phase enum table (phase 0, phase 1, etc...)
+CREATE TABLE cohort_phases (
+    id INTEGER PRIMARY KEY,
+    name TEXT NOT NULL,
+    UNIQUE (name)
+);
+
+-- Add cohort / phase relationship
+ALTER TABLE cohorts ADD COLUMN phase_id INTEGER REFERENCES cohort_phases;
```

Comment:
A couple comments on this:

1. If you create `cohort_phases` first, then this column can be part of the `CREATE TABLE` statement.
2. Can a cohort ever not be in a phase? I suspect this should be a `NOT NULL` column.

Reply:
Ah, you're right, cohort must be created with a phase as it is required